doc/hooks/xfce4-dev-tools.section.md

```diff
@@ -0,0 +1,5 @@
+# `xfce.xfce4-dev-tools` {#xfce4-dev-tools}
+
+This setup hook attempts to run `xdt-autogen` in `xdtAutogenPhase`, which is part of `preConfigurePhases`.
+
+{#dontUseXdtAutogenPhase} This behavior can be disabled by setting `dontUseXdtAutogenPhase` to `true`.
```

Comment:
For some reason `{#dontUseXdtAutogenPhase}` is rendered as-is for me though I also see this elsewhere, searching `{#` :joy:

<img src="https://github.com/user-attachments/assets/5770ce8e-2a09-49d9-b8b7-2f6f4c158218" width=50%>

Reply:
Fixed. Apparently, this needs to be a pseudo-link in the form `[]{#dontUseXdtAutogenPhase}` when it's not part of a heading.